membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -164,8 +164,9 @@ class TouchpointComponents(
 
   lazy val idapiService = new IdapiService(backendConfig.idapi, RequestRunners.futureRunner)
   lazy val tokenVerifierConfig = OktaTokenValidationConfig(
-    issuerUrl = conf.getString("okta.verifier.issuerUrl"),
-    audience = conf.getString("okta.verifier.audience"),
+    issuerUrl = OktaIssuerUrl(conf.getString("okta.verifier.issuerUrl")),
+    audience = Some(OktaAudience(conf.getString("okta.verifier.audience"))),
+    clientId = None,
```

Comment:
not sure why I needed a client id for the updated identity play auth library, but I didn't notice any issues leaving it as None so far.

Reply:
Have you tried asking Kelvin about this?

Reply:
good idea, it was added in this PR https://github.com/guardian/identity/pull/2399/files#diff-1ec0baf294cfe211cd46a240cd6d936ca0219600c1bf85bec53b1c1ee3fea773R19

I have asked on identity chat.